docs/writing-docs/mdx.md

```diff
@@ -459,7 +459,16 @@ If you're still encountering issues, we recommend reaching out to the community
 
 If you turned off inline rendering for your stories via the [`inline`](../api/doc-block-story.md#inline) configuration option, you would run into a situation where the associated controls are not updating the story within the documentation page. This is a known limitation of the current implementation and will be addressed in a future release.
 
-#### Learn more about Storybook documentation
+### The React version used is unexpected
```

Comment:
@JReinhold — Does this look like sufficient documentation?

Reply:
✨